modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,198 @@ The properties for broker-wide throughput quota balancing are configured at the
-  },
-  {
-    "group_name": "group_2",
-    "clients_prefix": "producer_group_multiple",
-    "quota": 20480
-  },
+"kafka_client_group_byte_rate_quota": [ 
+  { 
+    "group_name": "group_1", 
+    "clients_prefix": "producer_group_alone_producer", 
+    "quota": 10240 
+  }, 
+  { "group_name": "group_2", 
+    "clients_prefix": "producer_group_multiple", 
+    "quota": 20480 
+  }, 
 ]
 ----
+
+. Each client quota cluster property corresponds to a quota type in Kafka. Check the corresponding `rpk` arguments to use when setting the new quota values:
++
+|===
+| Cluster configuration property | `rpk cluster quotas` arguments
+
+| xref:reference:properties/cluster-properties.adoc#target_quota_byte_rate`[`target_quota_byte_rate`]                      
+| `--default client-id --add producer_byte_rate=<producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#target_fetch_quota_byte_rate[`target_fetch_quota_byte_rate`]                
+| `--default client-id --add consumer_byte_rate=<consumer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_admin_topic_api_rate[`kafka_admin_topic_api_rate`]                  
+| `--default client-id --add controller_mutation_rate=<controller-mutation-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`]          
+| `--name client-id-prefix=<prefix> --add producer_byte_rate=<group-producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`]    
+| `--name client-id-prefix=<prefix> --add consumer_byte_rate=<group-consumer-byte-rate-value>`
+
+|===
++
+The client throughput quotas set through the Kafka API apply per broker, so you must convert the cluster configuration values that were applied on a per-shard (logical CPU core) basis. For example, if you set `target_fetch_quota_byte_rate` to 100MBps/shard, and you run Redpanda on 16-core brokers, you can set the new consumer_byte_rate quota to 100 * 16 = 1600MBps.
+
+. Use `rpk cluster quotas alter` to set the corresponding client throughput quotas based on the Kafka API:
++
+[,bash]
+----
+rpk cluster quotas alter --name client-id-prefix=producer_group_alone_producer --add producer_byte_rate=<group-1-producer-byte-rate-value> 
+rpk cluster quotas alter --name client-id-prefix=producer_group_multiple --add producer_byte_rate=<group-2-producer-byte-rate-value> 
+----
++
+Replace the placeholder values with the new quota values, accounting for the conversion to per-broker limits, for example 10240 * broker core count = new quota.
+
+. Use `rpk cluster config` to remove the configuration-based quotas:
++
+[,bash]
+----
+rpk cluster config kafka_client_group_byte_rate_quota=
+----
+
+=== View throughput limits in Redpanda Console
+
+You can also use Redpanda Console to view enforced limits. In the menu, go to **Quotas**.
```

Comment:
@weeco Is there anything else with the Console view that needs to be called out here?

@mattschumpert It looks like the Console-specific content for this might be super light at the moment, so I chose to add it in this doc instead of creating a new standalone Console doc. However, if you have a suggestion for where it could go in the [existing Console docs](https://deploy-preview-592--redpanda-docs-preview.netlify.app/current/manage/console/) I can move it there too. I hesitate to add a screenshot here because I don't know how quickly it will become out of date.

Reply:
I think I agree.  @weeco  is actually working on this now and the screenshot may change this week as its finalized.